README.md

```diff
@@ -57,6 +57,27 @@ myFont.onload = evt => doSomeFontThings(evt);
 myFont.src = `./fonts/SourceCodeVariable-Roman.otf.woff2`;
 ```
 
+You can also pass a base64 string. In that case, you'll have to pass the filename as well, so the font type can de deducted from the extension:
+
+```js
+const myFont = new Font(`Adobe Source Code Pro`);
+
+// Grab file frop drop event or file upload
+const file = e.target.files[0];
+
+// Use FileReader to convert the file it to base64
+const reader = new FileReader();
+reader.readAsDataURL(file);
```

Comment:
Let's keep the data binary rather than converting to base64, since it'll end up in a binary [DataView](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView) anyway.
```suggestion
reader.readAsArrayBuffer(file);
```
https://developer.mozilla.org/en-US/docs/Web/API/FileReader/readAsArrayBuffer

Reply:
This doesn't work with `fetch` in `loadFont`. What'd be the best way to handle that?

Reply:
get the body as buffer: https://developer.mozilla.org/en-US/docs/Web/API/Body/arrayBuffer